### DISCUSSION TITLE
created custom bundle not loading ext js chnages on UI

### DISCUSSION THREAD

**Discussion by shubhangi101092** [pimcore/pimcore#16857] [Q&A / Support] [answered]
Hi ,

I have added one custom bundle inside bundles folder using pimcore bundle generate command ,bundle is installed and enabled and controller routes are working but when doing below change in startup.js its not loading in the UI, this '/bundles/custom/js/pimcore/startup.js', path is already added in customBundle.php inside getJsPaths method.

pimcore.registerNS("pimcore.plugin.CustomBundle");

pimcore.plugin.CustomBundle = Class.create(pimcore.plugin.admin, {
    getClassName: function () {
        return "pimcore.plugin.CustomBundle";
    },

    initialize: function () {
         alert(2);
        pimcore.plugin.broker.registerPlugin(this);
    },

    pimcoreReady: function (params, broker) {
         alert("CustomBundle ready!");
    }
});

var CustomBundlePlugin = new pimcore.plugin.CustomBundle();

Kindly suggest if anything i am missing .the pimcore version is 10.5

**Comment by AlephLau** [comment]


**Reply by shubhangi101092** [reply]
@AlephCode  yes its there,  Its automatically gets created when i run asset:install command,

**Comment by skoch98** [comment, marked as answer]
Hi @shubhangi101092 - you must also load the JS file via Event:

 - https://github.com/pimcore/demo/blob/11.x/src/EventListener/PimcoreAdminListener.php#L35
 - https://github.com/pimcore/demo/blob/11.x/config/services.yaml#L189

If this does not work either, add `?_dc=1` after the file path:
```
                $event->getPaths(),
                [
                    '/bundles/custom/js/pimcore/startup.js?_dc=1'
                ]
```

**Reply by skoch98** [reply]
And for development purposes I recommend to obtain the assets via symlink: `php bin/console assets:install public --symlink`

**Comment by shubhangi101092** [comment]
Thanks you so much @skoch98  after adding ?_dc=1  it worked out ! Hope this solution will help to many !
